### PR TITLE
Dynawo configuration

### PR DESCRIPTION
**Pull request by marcosmc**
**Please check if the PR fulfills these requirements** *(please use `'[x]'` to check the checkboxes, or submit the PR and then click the checkboxes)*
- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** *(Bug fix, feature, docs update, ...)*
feature

**What is the new behavior (if this is a feature change)?**
Required module to run dynawo. It defines the install directory of the dynawo simulator and defines if the temporary folder where the inputs are generated should be kept after the simulation.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>com.powsybl</groupId>
+        <artifactId>powsybl-parent</artifactId>
+        <version>1</version>
```
Comment (mathbagu) [inline pom.xml:16]: Change the parent version to the more recent one.

File: pom.xml [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+        <developer>
+            <name>Marcos de Miguel</name>
+            <email>demiguelm@aia.es</email>
+            <organization>Grupo AIA</organization>
+            <organizationUrl>http://www.aia.es</organizationUrl>
+        </developer>
+    </developers>
+
+    <modules>
+        <module>dynawo</module>
```
Comment (mathbagu) [inline pom.xml:43]: Could you remember me why do we have submodules?

File: pom.xml [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+                                <goals>
+                                    <goal>aggregate-jar</goal>
+                                </goals>
+                            </execution>
+                        </executions>
+                    </plugin>
+                </plugins>
+            </build>
+        </profile>
+    </profiles>
```
Comment (mathbagu) [inline pom.xml:78]: Do we really need this profile?

File: dynawo/pom.xml [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+                        <manifestEntries>
+                            <Automatic-Module-Name>com.powsybl.dynawo</Automatic-Module-Name>
+                        </manifestEntries>
+                    </archive>
+                </configuration>
+            </plugin>
+            <plugin>
+                <groupId>org.codehaus.gmavenplus</groupId>
+                <artifactId>gmavenplus-plugin</artifactId>
+            </plugin>
```
Comment (mathbagu) [inline dynawo/pom.xml:39]: As there is no groovy yet, remove this plugin activation

File: dynawo/src/main/java/com/powsybl/dynawo/simulator/DynawoConfig.java [hunk 5]
```diff
@@ -0,0 +1,42 @@
+package com.powsybl.dynawo.simulator;
```
Comment (mathbagu) [inline dynawo/src/main/java/com/powsybl/dynawo/simulator/DynawoConfig.java:1]: Don't forget to add the license header

File: dynawo/src/test/java/com/powsybl/dynawo/simulator/DynawoConfigTest.java [hunk 6]
```diff
@@ -0,0 +1,53 @@
+package com.powsybl.dynawo.simulator;
```
Comment (mathbagu) [inline dynawo/src/test/java/com/powsybl/dynawo/simulator/DynawoConfigTest.java:1]: Don't forget the license header

File: dynawo/src/test/java/com/powsybl/dynawo/simulator/DynawoConfigTest.java [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+
+    @After
+    public void tearDown() throws IOException {
+        fileSystem.close();
+    }
+
+    private void checkValues(DynawoConfig config, String homeDir, boolean debug) {
+        assertEquals(config.getHomeDir(), homeDir);
+        assertEquals(config.isDebug(), debug);
+    }
```
Comment (mathbagu) [inline dynawo/src/test/java/com/powsybl/dynawo/simulator/DynawoConfigTest.java:39]: This method is not needed: move the two assertions directly in the test case.
Put the expected value on the left: the prototype of the assertEquals method is `assertEquals(expected, actual)`

File: .github/workflows/maven.yml [hunk 8]
```diff
@@ -0,0 +1,34 @@
...
+      - name: Run SonarCloud analysis
+        if: matrix.os == 'ubuntu-latest'
+        run: >
+          mvn --batch-mode -Pjacoco verify sonar:sonar
+          -Dsonar.host.url=https://sonarcloud.io
+          -Dsonar.organization=powsybl-ci-github
+          -Dsonar.projectKey=com.powsybl:powsybl-dynawo
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
```
Comment (agnesLeroy) [inline .github/workflows/maven.yml:34]: Why is this file added?

File: dynawo/src/main/java/com/powsybl/dynawo/simulator/DynawoConfig.java (hunk 5 above)
Comment (agnesLeroy) [reply dynawo/src/main/java/com/powsybl/dynawo/simulator/DynawoConfig.java:7]: Beware that the copyright is 2020, not 2019

File: dynawo/src/test/java/com/powsybl/dynawo/simulator/DynawoConfigTest.java (hunk 6 above)
Comment (agnesLeroy) [reply dynawo/src/test/java/com/powsybl/dynawo/simulator/DynawoConfigTest.java:7]: Same comment about the year

File: pom.xml (hunk 1 above)
Comment (marcosmc) [reply pom.xml]: Done

File: pom.xml (hunk 3 above)
Comment (marcosmc) [reply pom.xml]: The profile is deleted

File: pom.xml (hunk 2 above)
Comment (marcosmc) [reply pom.xml]: The submodule is delected

File: dynawo/pom.xml (hunk 4 above)
Comment (marcosmc) [reply dynawo/pom.xml]: Done

File: dynawo/src/main/java/com/powsybl/dynawo/simulator/DynawoConfig.java (hunk 5 above)
Comment (marcosmc) [reply dynawo/src/main/java/com/powsybl/dynawo/simulator/DynawoConfig.java]: Done

File: dynawo/src/test/java/com/powsybl/dynawo/simulator/DynawoConfigTest.java (hunk 6 above)
Comment (marcosmc) [reply dynawo/src/test/java/com/powsybl/dynawo/simulator/DynawoConfigTest.java]: Done

File: dynawo/src/test/java/com/powsybl/dynawo/simulator/DynawoConfigTest.java (hunk 7 above)
Comment (marcosmc) [reply dynawo/src/test/java/com/powsybl/dynawo/simulator/DynawoConfigTest.java]: Done

File: .github/workflows/maven.yml (hunk 8 above)
Comment (marcosmc) [reply .github/workflows/maven.yml:34]: This file defines the CI job, I copied it from the old_master
